modules/openideal_idea/src/Plugin/Block/OpenidealIdeaTags.php

```diff
@@ -30,14 +30,21 @@ class OpenidealIdeaTags extends BlockBase {
   public function build() {
     $build = [];
     if ($node = $this->getEntity($this->getContexts())) {
+      $field = $node->bundle() == 'idea' ? 'field_idea_tags' : 'field_tags';
+      // In case when field is empty do not need to render anything at all.
+      if ($node->{$field}->isEmpty()) {
+        return [
+          '#cache' => ['#tags' => [$node->getCacheTags()]],
```

Comment:
Let's add cache metadata in the first line of build() function like this:
```
$build = [
  '#cache' => ['#tags' => [$node->getCacheTags()]],
];
```

Then, we can return $build in case of empty result.

Reply:
But on the first line of build() method $node variable isn't available.

Reply:
The idea is to avoid duplicated `$node->getCacheTags();`